packages/gateway/src/features/registration/utils.ts

```diff
@@ -64,3 +65,41 @@ export async function setCollectorForPrintInAdvance(
 
   return details
 }
+
+export type IPractitionerRoleHistory = (Resource & PractitionerRole)[]
```

Comment:
https://github.com/opencrvs/opencrvs-core/blob/fc2594d8e414816f41e00951fb06bb17af2b281a/packages/commons/src/fhir/practitioner.ts#L28
I think we could move these to commons, and use the types available here?